live-build/misc/ansible-roles/appliance-build.virtualization-common/tasks/main.yml

```diff
@@ -45,3 +45,34 @@
       domain=
       dns1=
       dns2=
+
+- name: Add the CLI user with home directory
+  user:
+    name: cli
+    comment: Delphix CLI User
+    uid: 65434
+    non_unique: yes
```

Comment:
[nit] This is a fairly unusual option to enable, but I think I understand why it's necessary (because all CLI users share the same UID). We might want to consider adding a comment clarifying this.

Reply:
that's right. Although I'll move this to the post-install hook of the debian package, as per @prakashsurya's comment below.